tests/test_ell1h.py

```diff
@@ -143,6 +144,17 @@ def test_J0613_STIG(toasJ0613, modelJ0613_STIG):
     f.fit_toas()
 
 
+def test_SINI_M2():
+    """Test SINI and M2 error.
+    """
+    SINI_par = simple_par.replace("H3 2.7507208E-7", "SINI 0.8")
+    with pytest.warns(None):
```

Comment:
What warning should be emitted? Please match on the message, not just to make the test better but also so I know what is being tested for.